packages/metaquot/metaquot.0.6.0/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "0.6.0"
```

Comment:
The linter complains about this line. We actually don't really put the version into the `opam` file (where the structure of the repository already gives to us the version).